lib/src/commands/OptionsProcessor.test.ts

```diff
@@ -43,14 +43,14 @@ describe('navigation options', () => {
       blurOnUnmount: false,
       popGesture: false,
       modalPresentationStyle: OptionsModalPresentationStyle.fullScreen,
-      animations: { dismissModal: { alpha: { from: 0, to: 1 } } },
+      animations: { dismissModal: { content: { alpha: { from: 0, to: 1 } } } },
```

Comment:
@mrousavy Modals don't have a content property. When declaring modal animations, the entire view of the modal is animated.

content is used to declare animations that operate on the views pushed into a stack.

We should keep the current API unchanged.